SourceCode/SmokeTest.Tests/SmokeTest.Tests.csproj

```diff
@@ -34,29 +34,32 @@
     <Reference Include="BCrypt.Net, Version=0.1.5073.29922, Culture=neutral, PublicKeyToken=6bcd409ee6bc9292, processorArchitecture=MSIL">
       <HintPath>..\packages\BCrypt-Official.0.1.109\lib\BCrypt.Net.dll</HintPath>
     </Reference>
-    <Reference Include="DbContextHelper, Version=3.0.0.2, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\RelativityDev.DbContextHelper.4.0.0.1\lib\net462\DbContextHelper.dll</HintPath>
+    <Reference Include="BCrypt.Net-Core, Version=1.6.0.0, Culture=neutral, PublicKeyToken=4d63bfbfdfd3c6e4, processorArchitecture=MSIL">
+      <HintPath>..\packages\BCrypt.Net-Core.1.6.0\lib\net452\BCrypt.Net-Core.dll</HintPath>
     </Reference>
-    <Reference Include="kCura, Version=10.1.199.4, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Relativity.Other.10.1.199.4\lib\net462\kCura.dll</HintPath>
+    <Reference Include="DbContextHelper, Version=5.0.0.1, Culture=neutral, processorArchitecture=MSIL">

```

Comment:
I thought we removed the DbContextHelper dependency from DevVM. @TommyBigNose Can you please remove the nuget package from the solution and compile and verify it's not being consumed anywhere.

Reply:
Updated, no longer using this, tested with updates on the removal